src/applications/appeals/10182/components/ContestableIssuesWidget.jsx

```diff
@@ -173,7 +175,7 @@ const ContestableIssuesWidget = props => {
   });
 
   const showNoIssues =
-    items.length === 0 && (!onReviewPage || (onReviewPage && inReviewMode));
+    showNoLoadedIssues && (!onReviewPage || (onReviewPage && inReviewMode));
 
   return (
     <>
```

Comment:
should `submitted={submitted}` be removed from here like it was for 995 & 996 so that it's 
`{showNoIssues && <NoIssuesLoadedAlert />}` on line 183